sound/soc/sof/sof-pci-dev.c

```diff
@@ -326,6 +326,7 @@ static int sof_pci_probe(struct pci_dev *pci,
 		dev_warn(dev, "warning: No matching ASoC machine driver found\n");
 	} else {
 		mach->mach_params.platform = dev_name(dev);
+		dev_dbg(dev, "ranjani %s\n", dev_name(dev));
```

Comment:
remove ranjani ?

Reply:
sorry, please ignore it. I find it was removed in following commit.

Reply:
@RanderWang thanks for pointing out. This should be removed in the first commit.